Author: Samuel Thibault <sthibault@debian.org>
Subject: Make hints/gnu.sh append to $ccflags rather than overriding them
Bug-Debian: http://bugs.debian.org/587901

Don't override possible extra $ccflags values given to Configure
on GNU/Hurd.

---
 hints/gnu.sh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/hints/gnu.sh b/hints/gnu.sh
index 2cfce54..c1ba2db 100644
--- a/hints/gnu.sh
+++ b/hints/gnu.sh
@@ -19,7 +19,7 @@ lddlflags='-shared'
 ccdlflags='-Wl,-E'
 
 # Debian bug #258618
-ccflags='-D_GNU_SOURCE'
+ccflags="-D_GNU_SOURCE $ccflags"
 
 # The following routines are only available as stubs in GNU libc.
 # XXX remove this once metaconf detects the GNU libc stubs.
-- 
tg: (c823880..) fixes/hurd-ccflags (depends on: upstream)
